docs/UserGuide.md

```diff
@@ -67,6 +67,10 @@ and flags(e.g. `-p`, `-D`, etc.) are **case-sensitive**.
 Adds an item to the list of tracked inventory.
 
 Format: `addi n/ITEM_NAME q/QUANTITY d/DESCRIPTION [t/TAG]…​ sp/SELL_PRICE cp/COST_PRICE`
+- `SELL_PRICE` is the price that the user sells the item for
+- `COST_PRICE` is the price that the user purchased the item for
```

Comment:
I think a better description would be 

`COST_PRICE` is the amount that it costs to produce per unit of the item